tests/test_hygiene.py

```
@@ -0,0 +1,381 @@
+    def test_heartbeat(self):
+        score, reasons = _score_noise("heartbeat", 0.5, "heartbeat")
+        assert score >= 0.7
+        assert "trivial_keyword" in reasons or "noisy_source" in reasons
+
+    def test_secret(self):
+        score, reasons = _score_noise("password = hunter2supersecret", 0.5, "")
+        assert score >= 0.9
+        assert any("secret" in r for r in reasons)
+
+    def test_valuable_content(self):
+        score, reasons = _score_noise("User prefers concise responses in English.", 0.7, "conversation")
```

Ruff (0.15.20)

[warning] Unpacked variable `reasons` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)